src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -119,12 +125,24 @@ public CommandBase turnOffBrake() {
         return new InstantCommand(() -> elevatorBrake.set(true), this);
     }
 
-    public CommandBase moveTicks(double ticks, double speed) {
+    public CommandBase moveTicks(double ticks, double speed, boolean checkStops) {
         CommandBase cmd = new FunctionalCommand(() -> {
             liftEncoder.reset();
         }, () -> {
+            double realSpeed = speed;
+            if (ticks < 0 && speed > 0) {
+                realSpeed *= -1;
+            }
+            if (checkStops) {
+                if (ticks < 0 && upperLimitSwitch.getAsBoolean()) {
+                    elevatorMotor.set(realSpeed);
+                } else if (ticks > 0 && lowerLimitSwitch.getAsBoolean()) {
+                    elevatorMotor.set(realSpeed);
+                }
```

Comment:
Shouldn't there be an else here to stop the motor?

Reply:
motor will never start